Examples/Docs/Sources/AsyncExamples/main.swift

```diff
@@ -86,7 +86,7 @@ private func changeStreams() throws {
         let inventory = db.collection("inventory")
 
         // Option 1: use next() to iterate
-        let next = inventory.watch(options: ChangeStreamOptions(fullDocument: .updateLookup))
+        let next = inventory.watch(options: ChangeStreamOptions(fullDocument: .other("blah")))
```

Comment:
these docs examples are meant to demonstrate basic behavior and will be included the [official docs page for change streams](https://docs.mongodb.com/manual/changeStreams/), so I think we should stick to `.updateLookup` rather than demonstrating the escape hatch.